kfs-ld/src/main/java/edu/arizona/kfs/module/ld/util/EreSweepBalanceHelper.java

```diff
@@ -0,0 +1,93 @@
+package edu.arizona.kfs.module.ld.util;
+
+import edu.arizona.kfs.module.ld.LaborConstants;
+
+public class EreSweepBalanceHelper {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(EreSweepBalanceHelper.class);
+	
+	private String accountNumber;
+	private String subAccountNumber = LaborConstants.EMPTY_SUB_ACCOUNT_STRING;
+	private String financialSubObjectCode = LaborConstants.EMPTY_SUB_OBJECT_STRING;
+	private String finObjectCode;
+	private String employeeId;
+	private String positionNbr;
+	
+	public EreSweepBalanceHelper(Object[] rowData) {
+		for (int i=0; i < rowData.length; ++i) {
+			if (rowData[i] != null) {
+				switch(i) {
+				case 0:
+					this.accountNumber = rowData[0].toString();
+					break;
+				case 1:
+					this.finObjectCode = rowData[1].toString();
+					break;
+				case 2:
+					this.employeeId = rowData[2].toString();
+					break;
+				case 3:
+					this.positionNbr = rowData[3].toString();
+					break;
+				case 4:
+					this.subAccountNumber = rowData[4].toString();
+					break;
+				case 5:
+					this.financialSubObjectCode = rowData[5].toString();
+					break;
+				default:
```

Comment:
Very good on the default case; I had a prof that would dock a semester's letter grade if we ever forgot to.